{{cookiecutter.project_slug}}/Makefile

```diff
@@ -65,7 +65,24 @@ check-lint-and-test-frontend:  ## Frontend Lint & Typecheck & Test
 	npm run lint-src; \
 	npm run typecheck; \
 	npm run test
-{% endif  %}
+{% endif %}
+
+backend-test:  ## Execute backend tests
+	cd backend; \
+	pip --disable-pip-version-check install --no-deps -r requirements/production.txt; \
+	pip --disable-pip-version-check install --no-deps -r requirements/tests.txt; \
+	pytest --cov=./ --cov-report html --ds=config.settings.test
+
+{% if cookiecutter.create_react_frontend == 'y' %}
+frontend-test:  ## Execute frontend tests
+	cd frontend; \
+	npm install --legacy-peer-deps; \
+	npm run test
```

Comment:
This is fine for now, but one should note that these targets will only work if you run them inside the container or install the dependencies locally. Since we are promoting running tests in the same container images, we should probably add targets to run them inside the container without having to shell into the container and then run `make backend-test`. For example:
```
backend-test:
	pod_name=$$(kubectl get pods -l app=django -o jsonpath="{.items[0].metadata.name}") && \
	kubectl exec $$pod_name -- make backend-tests
```

I will leave it up to come up with a solution that will work for the pipeline as well as running it locally. We can address this as a new issue by all means, just leave a comment in the Makefile.

Reply:
Yes, this will only work running in CI. To run locally, we also need some services up (like Postgres) and this is not 100% solved yet (I thought about some options, but didn't found the solution).